### ISSUE TITLE
Persistence is broken on version 3.8.23 and above with docker swarm

### ISSUE THREAD

**Issue by clgriffin** [bitnami/bitnami-docker-rabbitmq#191]
### Name and Version

3.8.23

### What steps will reproduce the bug?

Do a fresh docker deploy of 3.8.23 or above. Meaning, there is no running service or volume on the swarm.
Create a queue via the UI
Scale service to 0
Scale service to 1
Login to UI
Queue is gone ( not persisted)

Repeat above on 3.8.22 and it works fine.

### What is the expected behavior?

Expecting the same behavior as 3.8.22 and before.
Do a fresh docker deploy of 3.8.22 or below. Meaning, there is no running service or volume on the swarm.
Deploy docker service with 3.8.22
Create a queue via the UI
Scale Service to 0
Scale service to 1
Log back into the UI
Queue still exists (persisted)

### What do you see instead?

On startup, it does not appear to acknowledge there is persistent data.  Any version after 3.8.22 does not persist data on the volume in the swarm.

### Additional information

It only occurs on a fresh install with a new volume.
If you first deploy 3.8.22 and then deploy 3.8.23 and above on top of 3.8.22 with an existing volume, Persistence works.

**Comment by juan131** [comment]
Hi @clgriffin 

What mountpoint are you using for your data? It seems it that the recommended mountpoint for the data volume changed from `/bitnami` to `/bitnami/rabbitmq/mnesia` in that version. See:

- https://github.com/bitnami/bitnami-docker-rabbitmq/commit/fe3367e0fdae7d105b673aae6e32585b2ec58588#diff-18886f22711cf0e77b5b3449409c91b1d1c659110b50a21efa54ea8796a6b5cd
- https://github.com/bitnami/bitnami-docker-rabbitmq/blob/master/docker-compose.yml#L16

**Comment by clgriffin** [comment]
Yes... I am using /bitnami.  I will re-test using the mountpoint above and reply afterwards.
